File: php-laraval-app/app/Models/UserInfo.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\SoftDeletes;

class UserInfo extends Model
{
    use SoftDeletes;

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'avatar',
        'language',
        'timezone',
        'currency',
        'notification_method',
        'two_factor_auth',
        'marketing',
    ];

    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
        'deleted_at' => 'datetime',
    ];

    /**
     * User info relation to user model.
     */
    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class)->withTrashed();
    }
}

File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/UserInfo.cs
using System;
using System.Collections.Generic;

namespace Downstairs.Infrastructure.Persistence.Models;

public partial class UserInfo
{
    public long Id { get; set; }

    public long UserId { get; set; }

    public string? Avatar { get; set; }

    public string? Language { get; set; }

    public string? Timezone { get; set; }

    public string? Currency { get; set; }

    public string? NotificationMethod { get; set; }

    public string TwoFactorAuth { get; set; } = null!;

    public sbyte? Marketing { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public DateTime? DeletedAt { get; set; }
}
